applogic/src/api/message_list_cubit.rs

```diff
@@ -42,6 +42,8 @@ pub struct MessageListState {
 #[frb(ignore)]
 #[derive(Debug, Default)]
 struct MessageListStateInner {
+    /// Whether the chat the messages are in is a connection chat
+    is_connection_chat: Option<bool>,
```

Comment:
I am not very happy to communicate this particular chat type like this. However, adding it to each message seems like too much overhead. And communicating `UiChatType` requires loading the user profile, which is not needed here.

Reply:
It is extracted from db, so it is not const, but dynamic, however it is read-only.